api/src/main/java/org/pmiops/workbench/reporting/ReportingVerificationServiceImpl.java

```diff
@@ -62,74 +64,54 @@ public boolean verifyAndLog(ReportingSnapshot reportingSnapshot) {
     final StringBuilder sb =
         new StringBuilder(
             String.format("Verifying Snapshot %d:\n", reportingSnapshot.getCaptureTimestamp()));
+
     sb.append("Table\tSource\tDestination\tDifference(%)\n");
-    Level detailsLogLevel = Level.INFO;
-    boolean verified = true;
-    for (final ReportingUploadResult result : uploadDetails.getUploads()) {
-      if (!verifyCount(
-          result.getTableName(), result.getSourceRowCount(), result.getDestinationRowCount(), sb)) {
-        detailsLogLevel = Level.WARNING;
-        verified = false;
-      }
-    }
-    logger.log(detailsLogLevel, sb.toString());
+
+    // fails-fast due to allMatch() so logs may be incomplete on failure
```

Comment:
I do not understand why the logs would be incomplete.

Reply:
verifyCount() adds logs to `sb` when it executes.  If it returns false for a row, it will never get executed for subsequent rows.

Previously, it would always get executed for all rows (continuing after failures)

I think it's desirable to fail-fast, because a partial reporting run is not useful.